### ISSUE TITLE
az blueprint assignment create from json parameter file

### ISSUE THREAD

**Issue by Linaa** [Azure/azure-cli-extensions#2248]
## Describe the bug

**Command Name**
`az blueprint assignment create
Extension Name: blueprint. Version: 0.1.1.`

**Errors:**
```
Unable to build a model: Unable to deserialize to object: type, AttributeError: 'str' object has no attribute 'get', DeserializationError: Unable to deserialize to object: type, AttributeError: 'str' object has no attribute 'get'
Traceback (most recent call last):
python3.8/site-packages/msrest/serialization.py, ln 1416, in _deserialize
    found_value = key_extractor(attr, attr_desc, data)
...
    return data.get(found_key)
msrest.exceptions.SerializationError: Unable to build a model: Unable to deserialize to object: type, AttributeError: 'str' object has no attribute 'get', DeserializationError: Unable to deserialize to object: type, AttributeError: 'str' object has no attribute 'get'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az blueprint assignment create --subscription {} --name {} --location {} --identity-type {} --description {} --parameters {}`

## Environment Summary
```
macOS-10.15.6-x86_64-i386-64bit
Python 3.8.5
Installer: HOMEBREW

azure-cli 2.10.1 *

Extensions:
azure-devops 0.12.0
subscription 0.1.4
blueprint 0.1.1

```

**Comment by yonzhan** [comment]
blueprint

**Comment by yungezz** [comment]
cc @houk-ms for awareness since this is typical code error.

**Comment by fengzhou-msft** [comment]
The issue is fixed in blueprint 0.2.0, please update and retry.
